modules/apps/foundation/frontend-theme/space-theme/src/templates/portal_pop_up.ftl

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title}</title>
+
+	<@liferay_util["include"] page=top_head_include />
+</head>
+
+<body class="portal-popup ${css_class}">
+		<button id="modalBtn" class="button">Open Modal</button>
```

Comment:
Indentation should only be 4 spaces/1 tab.

Reply:
Use a language key so the text can be localized instead of being hardcoded as just English.